packages/agents/src/mcp/index.ts

```diff
@@ -210,8 +210,24 @@ export abstract class McpAgent<
    */
   private _agent: Agent<Env, State>;
 
-  get mcp() {
-    return this._agent.mcp;
+  get mcpClientManager() {
+    return this._agent.mcpClientManager;
+  }
```

Comment:
This bit is technically breaking for anyone who's using the client manager, but I don't think anyone is and it's not documented.